views/js/test/mathEntryInteraction/data/qti.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assessmentItem
+        xmlns="http://www.imsglobal.org/xsd/imsqti_v2p1"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:m="http://www.w3.org/1998/Math/MathML"
+        xmlns:pci="http://www.imsglobal.org/xsd/portableCustomInteraction"
+        xmlns:html5="html5"
+        xsi:schemaLocation="http://www.imsglobal.org/xsd/imsqti_v2p1 http://www.imsglobal.org/xsd/qti/qtiv2p1/imsqti_v2p1.xsd"
+        identifier="i1472560050505106" title="Item_2 3" adaptive="false" timeDependent="false">
+  <responseDeclaration cardinality="single" identifier="RESPONSE" baseType="string"/>
+  <outcomeDeclaration cardinality="single" baseType="float" identifier="SCORE"/>
+  <stylesheet href="mathEntryInteraction/runtime/mathquill/mathquill.css" title="" type="text/css" media="all" />
+  <stylesheet href="mathEntryInteraction/runtime/css/mathEntryInteraction.css" title="" type="text/css" media="all" />
+  <itemBody>
+    <div class="grid-row">
+      <div class="col-12">
+        <customInteraction responseIdentifier="RESPONSE">
+          <pci:portableCustomInteraction customInteractionTypeIdentifier="mathEntryInteraction" hook="mathEntryInteraction/runtime/mathEntryInteraction.js">
+            <pci:responseSchema href="http://imsglobal.org/schema/json/v1.0/response.json"/>
+            <pci:resources location="http://imsglobal.org/pci/1.0.15/sharedLibraries.xml">
+              <pci:libraries>
+                <pci:lib id="IMSGlobal/jquery_2_1_1"/>
```

Comment:
other requires libs seem to be missing: is this sample xml file still up-to-date ?